patientsearch/src/js/components/Info.js

```diff
@@ -0,0 +1,131 @@
+import React from 'react';
+import { makeStyles} from '@material-ui/core/styles';
+import Button from '@material-ui/core/Button';
+import Typography from '@material-ui/core/Typography';
+import {imageOK, sendRequest} from './Utility';
+import theme from '../context/theme';
+
+const useStyles = makeStyles({
+    container: {
+        textAlign: "center",
+        marginLeft: theme.spacing(2),
+        marginRight: theme.spacing(2),
+        marginTop: theme.spacing(24),
+        display: "flex",
+        flexDirection: "column",
+        alignItems: "center",
+        minHeight: "100",
+        flex: "1 0 auto",
+        paddingBottom: theme.spacing(4)
+    },
+    button: {
+        width: "240px",
+        borderWidth: "2px",
+        marginTop: theme.spacing(3)
+    },
+    info: {
+        marginTop: theme.spacing(3),
+        maxWidth: "640px"
+    },
+    title: {
+        fontSize: "1.25rem",
+        lineHeight: "1.5"
+    }
+});
+
+export default function Info() {
+    const classes = useStyles();
+    const [setting, setSetting] = React.useState({});
+    const [siteID, setSiteID] = React.useState("");
+    const [initialized, setInitialized] = React.useState(false);
+    const SYSTEM_TYPE_STRING = "SYSTEM_TYPE";
+    //config name for site login HTML string
+    const SITE_INFO_STRING = "SITE_LOGIN_INFO";
```

Comment:
We'll have to see how variable Bill wants this to be, but if it's fixed for each site, it'd be more convenient for me to deploy/configure if it were mapped to site codes (like `siteNameMappings`)